helm-charts/images/hub/Dockerfile

```diff
@@ -18,8 +18,9 @@ RUN pip install --no-cache git+https://github.com/yuvipanda/jupyterhub-configura
 # z2jh 1.2.x ships with kubespawner 1.1.0, so we just do a little bump
 RUN pip install --no-cache --upgrade jupyterhub-kubespawner==1.1.2
 
-# Brings in https://github.com/jupyterhub/oauthenticator/pull/498
-RUN pip install --no-cache --upgrade git+https://github.com/yuvipanda/oauthenticator@7f3fdc0a14d06f1a081c23b1ceb7060a940d11f8
+# Brings in https://github.com/jupyterhub/oauthenticator/pull/498 and
+# https://github.com/jupyterhub/oauthenticator/pull/504
+RUN pip install --no-cache --upgrade git+https://github.com/jupyterhub/oauthenticator@60ddc4fe3ab06b19c6876992679c1fd2ec1c64dd
```

Comment:
Again, let's capture somewhere that we are running from a specific commit instead of a released version.
IIRC from a recent Slack conversation, @GeorgianaElena was interested to push forward on the release for this package, am I correct?

Reply:
I just opened https://github.com/2i2c-org/infrastructure/issues/1319 @damianavila. Sorry for taking so long, I somehow missed the notification.